src/index.html

```diff
@@ -1,14 +1,84 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;900&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="header">
+      <div class="list__logo">
+        <a href="#header" class="logo__link">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item ">
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item ">
```
don't add empty lines between parent and child element